packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -749,14 +742,14 @@ spec:
           template: |
             devices:
             - networkName: {{ .vcenter.network }}
-            {{ if .controlPlane.network.nameservers -}}
-              nameservers:
-              {{- range .controlPlane.network.nameservers }}
-            - {{ . }}
+              {{ if .controlPlane.network.nameservers -}}
+                nameservers:
```

Comment:
Hi Nick, are you intensionally adding 2 spaces before `nameservers` or it's a typo?

Reply:
This is intentional. This whole block was not correctly intended previously.

Reply:
Thank you @tenczar , though it looks strange that line 746 is not align with 744 and 748, suppose it should also work.
```
 744           - networkName: {{ .vcenter.network }}
 745             {{ if .controlPlane.network.nameservers -}}
 746               nameservers:
 747               {{- range .controlPlane.network.nameservers }}
 748             - {{ . }}
 749               {{- end }}
 750             {{- end }}
```